project/webapp/view/fragments/ProductDetails.fragment.xml

```diff
@@ -0,0 +1,189 @@
+<core:FragmentDefinition
+  xmlns="sap.m"
+  xmlns:l="sap.ui.layout"
+  xmlns:f="sap.ui.layout.form"
+  xmlns:core="sap.ui.core">
+  <VBox visible="{= !${editModel>/isEditMode}}">
+    <f:SimpleForm id="productDetails"
```

Comment:
It doesn't make sense to create two different SimpleForm for edit and read scenarios, 
You can just use two controls under one label and manage their visibility.
<Label>
<Text visible="{= !${editModel>/isEditMode}}" ...>
<Input visible="{editModel>/isEditMode}" ...>